src/shared/components/Select/Select.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { forwardRef } from 'react'
+import InputBase, { InputBaseProps } from '../InputBase'
+import { SelectButton, SelectMenu, SelectOption, SelectWrapper, StyledLabelText } from './Select.style'
+import { useSelect, UseSelectStateChange } from 'downshift'
+import Icon from '../Icon'
+
+export type SelectedItem = {
+  value: string
+  name: string
+} | null
+
+export type SelectProps = {
+  onChange?: (changes: UseSelectStateChange<SelectedItem>) => void
+  value?: SelectedItem
+  items: SelectedItem[]
+  placeholder?: string
+} & InputBaseProps
+
+const SelectComponent: React.ForwardRefRenderFunction<HTMLDivElement, SelectProps> = (props, ref) => {
+  const { label = '', items, placeholder = 'Select option', error, warning, value, disabled, onChange } = props
```

Comment:
I think we're missing the `helperText` here. Maybe we extract `...inputBaseProps` for the values that we don't use in this component and then pass that to `InputBase`? Currently I think this will only be `warning` and `helperText`